tests/EvmAcceptanceTests/test/ParentChild.js

```diff
@@ -42,10 +45,20 @@ describe("Parent Child Contract Functionality", function () {
       childContract = new web3.eth.Contract(hre.artifacts.readArtifactSync("ChildContract").abi, childContractAddress, {
         from: owner.address
       });
-
       expect(await childContract.methods.read().call()).to.be.eq(ethers.BigNumber.from(CHILD_CONTRACT_VALUE));
     });
 
+    it("Should create a transaction trace after child creation", async function () {
+      const METHOD = "debug_traceTransaction";
+
+      await helper.callEthMethod(METHOD, 1, [installedChild.hash], (result, status) => {
+        hre.logDebug(result);
+
+        assert.equal(status, 200, "has status code");
+        assert.isString(result.result, "Expected to be populated");
```

Comment:
couldn't it be more specific? using [method_property](https://www.chaijs.com/api/assert/#method_property) for example?